README.md

```diff
@@ -55,166 +56,118 @@ package:
 - Build system: [Make] or [Ninja]
-- for developers: Cython version 0.21 or higher (e.g., `pip3 install cython`)
-- Sphinx and it's bootstrap theme (e.g., `pip3 install sphinx sphinx_bootstrap_theme`)
-
-## Installation instructions
-
-### Installing NetworKit via pip
-
-    pip3 install [--user] networkit
-
-Depending on your system, you might need to install python3-tk.
-
-
-### Building NetworKit as a Python Module from source
-
-Run the script `setup.py` with the following options:
-
-	python3 setup.py build_ext [-jX]
-
-The script will call `cmake` and `ninja` (`make` as fallback) to compile NetworKit as a library, build the extensions and copy it to the top folder. By default, NetworKit will be built with the amount of available cores in optimized mode. It is possible the add the option `-jN` the number of threads used for compilation.
-The setup script provides more functionality and can be used with pip as well:
-
-	pip3 install -e .
-
-will compile NetworKit, build the extensions and on top of that temporarily install NetworKit so that it is available on the whole system. `pip3 uninstall networkit` will remove networkit.
-
-### `jupyterhub`: An interactive environment for working with NetworKit
+- Cython version 0.21 or higher (e.g., `pip3 install cython`)
 
-To check that everything works as expected, open a python terminal and run the example from the beginning of this document or the following lines:
+# Usage example
 
-    >>> import networkit
-    >>> G = networkit.Graph(5)
-    >>> G.addEdge(0,1)
-    >>> G.toString()
+In this example we generate a random hyperbolic graph with 100k nodes
+and we compute its communities with the PLM method:
 
-Additionally, we recommend that you familiarize yourself with NetworKit through experimenting with the interactive IPython Notebook `NetworKit_UserGuide.ipynb` located in the folder `Doc/Notebooks`. The user guide also introduces a large portion of NetworKit functionality with usage examples. To display and work with these notebooks, you have to install jupyterhub and start a local notebook server from the terminal with:
+    >>> import networkit as nk
+    >>> g = nk.generators.HyperbolicGenerator(1e5).generate()
+    >>> communities = nk.community.detectCommunities(g, inspect=True)
+    PLM(balanced,pc,turbo) detected communities in 0.14577102661132812 [s]
+    solution properties:
+    -------------------  -----------
+    # communities        4536
+    min community size      1
+    max community size   2790
+    avg. community size    22.0459
+    modularity              0.987243
+    -------------------  -----------
 
-	jupyterhub --no-ssl
+For more interactive examples you can have a look at our [notebooks]-section, especially the [Networkit UserGuide]. Note: To view the computed output from the notebooks, it is recommended to use ![JupyterHub](https://pypi.org/project/jupyterhub/) (installed for example via pip3 or conda). This requires the prior installation of NetworKit.
```

Comment:
IMHO, this should be put first (above the inline example). Also: JupyterHub or just plain jupyter-notebook (the notebook server)? JupyterHub is probably much harder to set up!